nav: {obs_avoid_dist: 250, obs_rot_ang: 0.1, obs_rot_lin: 0.3, obs_st_lin: 0.2, obs_stop_dist: 80,
  rot_ang_vel: 0.2, rot_lin_vel: 0.4, st_lin_vel: 0.4}
